myprotein

```diff
@@ -27,6 +35,26 @@ def main():
     for i in flavours:
         for k in sizes:
             get_price(id, i['id'], pouches[0]['id'], k['id'])
+   
+    print()
+
+    if args.vouchers:
+        get_all_vouchers()
+
+
+def get_all_vouchers():
+    print('Vouchers:')
+    page = requests.get(VOUCHER_URL)
+    soup = bs4.BeautifulSoup(page.content, 'html.parser')
+    containers = soup.find_all('div', class_='voucher-container')
```

Comment:
Any reason for using `find_all` instead of `select`? 

[Select](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#css-selectors) lets you use CSS selectors. This would become `soup.select('div.voucher-container')`.